packages/safe-core-sdk/README.md

```diff
@@ -162,28 +159,134 @@ await executeTxResponse.transactionResponse?.wait()
 
 All the signatures used to execute the transaction are now available at `safeTransaction.signatures`.
 
-## API Reference
+## Safe Factory API Reference <a name="factory-api"></a>
 
 ### create
-Returns an instance of the Safe Core SDK connected to the `safeAddress`.
+
+Returns an instance of the Safe Factory.
 
 ```js
-const safeSdk = await Safe.create({ ethAdapter, safeAddress })
+import { SafeFactory } from '@gnosis.pm/safe-core-sdk'
+
+const safeFactory = await SafeFactory.create({ ethAdapter })
 ```
 
-The property `contractNetworks` can be added to provide the Safe contract addresses in case the SDK is used in a network where the Safe contracts are not deployed.
+* The property `isL1SafeMasterCopy`
```

Comment:
"The `isL1SafeMasterCopy` property"

Would "flag" make more sense instead of "property"?

Reply:
There are other places where "flag" could be used instead as well in this file.